plugins/fcrdns

```diff
@@ -168,35 +167,35 @@ sub is_valid_localhost {
         $self->adjust_karma(1);
         $self->log(LOGDEBUG, "pass, is localhost");
         return 1;
-    };
+    }
 
     my $rh = $self->qp->connection->remote_host;
-    if ($rh && lc $self->qp->connection->remote_host eq 'localhost') {
-        $self->log(LOGDEBUG, "pass, remote_host is localhost");
-        return 1;
-    };
+    return 0 if ! $rh;
+    return 0 if lc $self->qp->connection->remote_host ne 'localhost';
 
     $self->adjust_karma(-1);
     $self->log(LOGINFO, "fail, not localhost");
```

Comment:
Are we subtracting karma from all remote hosts that are not localhost?  Or am I reading that wrong?